gtsam/geometry/tests/testRot3.cpp

```diff
@@ -661,6 +661,21 @@ TEST(Rot3, ClosestTo) {
   EXPECT(assert_equal(expected, actual.matrix(), 1e-6));
 }
 
+/* ************************************************************************* */
+TEST(Rot3, angle) {
```

Comment:
Hmmm. This is not a good test case. How do we know its correct? Simpler case?

Reply:
That's a good point. I'll come up with a more contrived example which we know is correct.